pilot/pkg/model/envoyfilter.go

```diff
@@ -162,11 +163,18 @@ func proxyMatch(proxy *Proxy, cp *EnvoyFilterConfigPatchWrapper) bool {
 	return true
 }
 
-func (efw *EnvoyFilterWrapper) Key() string {
+// Returns all the envoyfilters that wrapped.
+func (efw *EnvoyFilterWrapper) Keys() []string {
```

Comment:
it would be good to add the Envoyfilter part to cluster as well, maybe split out from this PR?

We can also add an integration test that applies the EnvoyFilter to a single workload, and maybe doesn't even really do anything - but it would trigger the assertion failures and act as a test maybe

Reply:
Yes. I think it is better to split it.